src/email/utils/sesOneEmail.ts

```diff
@@ -20,7 +20,7 @@ export const sendOneByOneEmail = (payload: IProps) => {
       // CcAddresses: ['recipient3@example.com'],
       ToAddresses: payload.to,
     },
-    ConfigurationSetName: 'sonu',
+    ConfigurationSetName: 'emailRules',
```

Comment:
changed configuration set's name to emailRules. so that email from boossti.com domain can be tracked. @appdatasoft